test/hdfs_test.py

```diff
@@ -272,7 +274,13 @@ def test_with_write_success(self):
 
 
 @attr('minicluster')
-class HdfsTargetTests(MiniClusterTestCase):
+class HdfsTargetTests(MiniClusterTestCase, BaseTargetTestCase):
```

Comment:
I think this won't work actually (because nosetest is buggy). Can you veryify that the attribute is still honored? I mean, does this still *only* run when running tox with either the `cdh` or `hdp` environment and not `nonhdfs`?